EventualConsistencyDemo/Views/Reviews/Movie.cshtml

```diff
@@ -2,30 +2,40 @@
 @{
     ViewData["Title"] = Model.Movie.Title;
 }
-<script>
 
-    $(document).ready(function() {
-
-        var newDate = new Date();
+@section Scripts{
+    <script id="review-template" type="text/x-custom-template">

```

Comment:
Ah, da's beter... Thx!

Reply:
Ik heb het zelf ook gehad.. (en zit er nog midden in al die appends)... Echt drama.. Op deze manier templates definieren is het veul makkelijker...